server/src/modules/maps/maps.service.ts

```diff
@@ -193,10 +188,6 @@ export class MapsService {
 
     private storeMapFile(mapFileBuffer, mapModel) {
         const fileName = `maps/${mapModel.name}.bsp`;
-        if (appConfig.storage.useLocal) {
```

Comment:
And remove `useLocal` from the config.

Reply:
[fixed](https://github.com/momentum-mod/website/pull/624/commits/9fb93c86be2d1183365dec5fe1193509989fd89f)